index.html

```diff
@@ -0,0 +1,434 @@
+              <div class="team-layout__name">Rodney Stratton</div>
+              <div class="team-layout__job">Consultant</div>
+              <div class="team-layout__socials">
+                <img src="./images/instragram.svg" alt="facebook-icon" />
+                <img src="./images/twitter.svg" alt="instragram-icon" />
+                <img src="./images/facebook.svg" alt="twitter-icon" />
+              </div>
+            </div>
+          </div>
+          <div class="team-layout__card">
+            <div class="team-layout__container-img">
+              <img
+                class="team-layout__img"
+                src="./images/cover (2).png"
+                alt="employee"
+              />
+            </div>
+            <div class="team-layout__info">
+              <div class="team-layout__name">Avie Beaton</div>
+              <div class="team-layout__job">CO Founder</div>
+              <div class="team-layout__socials">
+                <img src="./images/instragram.svg" alt="facebook-icon" />
+                <img src="./images/twitter.svg" alt="instragram-icon" />
+                <img src="./images/facebook.svg" alt="twitter-icon" />
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="contact-layout">
+        <div class="contact-layout__contents">
+          <div class="contact-layout__sub-title">Contact us</div>
+          <div class="contact-layout__title">Make an appointment</div>
+          <div class="contact-layout__input-container">
+            <div class="contact-layout__input-container1">
+              <input
+                class="contact-layout__input"
+                type="text"
+                placeholder="Full Name"
+              />
+              <input
+                class="contact-layout__input"
+                type="text"
+                placeholder="example@gmail.com"
+              />
+            </div>
+            <div class="contact-layout__input-container1">
+              <select class="contact-layout__input">
+                <option value="">Please Select</option>
+                <option value=""></option>
+              </select>
+              <select class="contact-layout__input">
+                <option value="en">4:00 Available</option>
+                <option value=""></option>
+              </select>
+            </div>
+            <div class="contact-layout__input-message">
+              <textarea
+                class="contact-layout__input-messages"
+                name=""
+                id=""
+                cols="30"
+                rows="10"
+              >
+Message</textarea
+              >
+            </div>
+            <div>
+              <button class="button button--link">Book an appointment</button>
+            </div>
+          </div>
+        </div>
+        <div class="contact-layout__agency">
+          <div class="contact-layaut__agency-content">
```

Comment:
You have a typo here with `contact-layaut__agency-content` instead of `contact-layout__agency-content`